R/fastMapGenerate.R

```diff
@@ -23,7 +23,7 @@
 #' @examples
 #' \dontrun{
 #' fastMapGenerate("hgnc_complete_set.txt", "symbol", "uniprot_ids",
-#'                 type = "UniProt", outName = "fastMapUniProt.rds")
+#'                 type = "UniProt", outputName = "fastMapUniProt.rds")
 #' }
```

Comment:
Yeah - this isn't the first time, I saw you catch the documentation for `fastmap` --> `fastMap`. What's the best way to alert him? Could I tag him directly to this comment? @RicardoRam